From 952ee2e7ae00129029db94d500e2f39c3c0256cf Mon Sep 17 00:00:00 2001
From: Masami Hiramatsu <mhiramat@kernel.org>
Date: Thu, 31 Oct 2019 21:23:00 +0900
Subject: [PATCH] selftests: breakpoints: Fix a typo of function name
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

commit-author Masami Hiramatsu <mhiramat@kernel.org>
commit 5b06eeae52c02dd0d9bc8488275a1207d410870b

Since commit 5821ba969511 ("selftests: Add test plan API to kselftest.h
and adjust callers") accidentally introduced 'a' typo in the front of
run_test() function, breakpoint_test_arm64.c became not able to be
compiled.

Remove the 'a' from arun_test().

Fixes: 5821ba969511 ("selftests: Add test plan API to kselftest.h and adjust callers")
        Reported-by: Jun Takahashi <takahashi.jun_s@aa.socionext.com>
        Signed-off-by: Masami Hiramatsu <mhiramat@kernel.org>
        Cc: Kees Cook <keescook@chromium.org>
        Reviewed-by: Kees Cook <keescook@chromium.org>
        Signed-off-by: Shuah Khan <skhan@linuxfoundation.org>
(cherry picked from commit 5b06eeae52c02dd0d9bc8488275a1207d410870b)
        Signed-off-by: Marcin Wcisło <marcin.wcislo@conclusive.pl>
---
 tools/testing/selftests/breakpoints/breakpoint_test_arm64.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/testing/selftests/breakpoints/breakpoint_test_arm64.c b/tools/testing/selftests/breakpoints/breakpoint_test_arm64.c
index ab59d814341a..8a53680a899b 100644
--- a/tools/testing/selftests/breakpoints/breakpoint_test_arm64.c
+++ b/tools/testing/selftests/breakpoints/breakpoint_test_arm64.c
@@ -118,7 +118,7 @@ static bool set_watchpoint(pid_t pid, int size, int wp)
 	return false;
 }
 
-static bool arun_test(int wr_size, int wp_size, int wr, int wp)
+static bool run_test(int wr_size, int wp_size, int wr, int wp)
 {
 	int status;
 	siginfo_t siginfo;
-- 
2.43.5

